### PR TITLE
[JS] Topic: Tutorial for JavaScript Operators Fixed #202

### LINKED ISSUE #202: JS: Operators in JavaScript
# Operators in JavaScript

### Directory

```
Topics/5.Syntax/2.Operators
```

### Description

* Talk about the various operators in JavaScript.
* Arithmetic, Assignment, Increment and Decrement, Comparison, Logical, Bitwise.
* Explain operators types through examples.

### Hints

* Check out the [W3C tutorial](https://www.w3schools.com/js/js_operators.asp)

## Formats
- [x] Document
- [ ] Video
- [ ] Audio

### PR DESCRIPTION
**Pull request by Himanshu664**
## Description 📜

Created a tutorial for operators in JavaScript.
Fixes #202 

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Himanshu664** [comment]
@paramsiddharth as you suggested in todays meet I have close my previous PR,and created a fresh PR made changes as you told me.

File: Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+Arithmetic operators take numerical values as operands and evaluates it to a single numerical value. Some of the arithmetic operators are -
+
+
+* **Addition(+)** - It adds the numerical operands and is also used for string concatenation.It will add numerical and string operand to a number if the string can be converted to a number. Else it concatenates them.
+
+    **Example:**  
+    ```js
+                   var x = 10;      
+                   var y = 4;       
+                   alert(x + y);
```
Comment (paramsiddharth) [inline Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md:18]: The unwanted indentation is still there!

File: Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+
+
+* **Addition(+)** - It adds the numerical operands and is also used for string concatenation.It will add numerical and string operand to a number if the string can be converted to a number. Else it concatenates them.
+
+    **Example:**  
+    ```js
+                   var x = 10;      
+                   var y = 4;       
+                   alert(x + y);  
+		       0utput : 14
```
Comment (paramsiddharth) [inline Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md:19]: It should be Output, not 0utput!
Also, the output should be outside the syntax-highlighted code-block.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]

**Comment by Himanshu664** [comment]
@paramsiddharth Please review changes.

**Review by paramsiddharth — APPROVED** [review]
@Creatoon @dakshsethi @s4-2 Please review further.

**Review by s4-2 — APPROVED** [review]

**Comment by s4-2** [comment]
@Himanshu664 very well structured. I have reviewed and approved it. Keep it up:-)

**Review by dakshsethi — CHANGES_REQUESTED** [review]

**Pull request merged**